src/app/commands/cmd_sprite_properties.cpp

```diff
@@ -40,6 +44,95 @@ namespace app {
 
 using namespace ui;
 
+class TilesetListItem : public ui::ListItem {
+public:
+  TilesetListItem(const doc::Tileset* tileset, doc::tileset_index tsi)
+    : ListItem(app::tileset_label(tileset, tsi)),
+    m_buttons(new HBox()) {
```

Comment:
There is an extra indentation in the ctor block:
```suggestion
  TilesetListItem(const doc::Tileset* tileset, doc::tileset_index tsi)
    : ListItem(app::tileset_label(tileset, tsi))
    , m_buttons(new HBox())
  {
```